pkg/tokens/builtin/zoneingress/issuer.go

```diff
@@ -0,0 +1,91 @@
+package zoneingress
+
+import (
+	"github.com/dgrijalva/jwt-go"
+	"github.com/pkg/errors"
+)
+
+type Token = string
+
+type Identity struct {
+	Name string
```

Comment:
please add zone or tags if we change to tags but only if `kuma.io/zone` is required?